### ISSUE TITLE
DQ script can't handle yyyy format birthdates

### ISSUE THREAD

**Issue by palmjulia** [medizininformatik-initiative/Projectathon6-smith2#4]
Our data contains only only years in `cohort$birthdate`, not month or day (e.g. `1967`). 

The DQ script throws an error [line 99](https://github.com/medizininformatik-initiative/Projectathon6-smith2/blob/DQ/smith_dq_report.RMD#L99) because `as.Date()` says `character string is not in a standard unambiguous format`. 

You'd probably have to check for the kind of date format provided but I don't know how this is then best handled from a DQ perspective. Either add a day and a month or create new rules for year-only dates?

**Comment by chhdraeger** [comment]
As as.Date doesn't coerse to NA I replaced it with parse_date_time from the `lubridate` package. It should now correctly convert anything from `YYYY, YYYY-MM or YYYY-MM-DD` to the datetime format required by `dataquieR`. 
Anything following a different format will be replaced with the missing-code `9991200`. 
I also added calculated age based on lubridates functions. It is now included as one of the plotted distributions in the quaility report.
